clients/client-python/tests/integration/init/launch.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# Copyright 2024 Datastrato Pvt Ltd.
+# This software is licensed under the Apache License version 2.
+#
+cd "$(dirname "$0")"
+
+playground_dir="$(dirname "${BASH_SOURCE-$0}")"
+playground_dir="$(cd "${playground_dir}">/dev/null; pwd)"
+isExist=`which docker`
+if [ ! $isExist ]
+then
+  echo "ERROR: No docker service environment found, please install docker first."
+  exit
```

Comment:
We should add exit code except for 0 here, or the exit code here will come from `echo`, which would be 0, and it won't be caught by function in `build.kts.gradle`

Reply:
Done.